cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/replication/ReplicatorBuilder.java

```diff
@@ -317,6 +376,30 @@ public E withId(int id) {
     }
 
     /**
+     * <p>
+     *     Sets the IAM API key to use for authenticating requests.
+     * </p>
+     * <p>
+     *     Note: the replicator will only use IAM to authenticate requests and will not attempt to
+     *     use cookies or basic authentication. This means that the userinfo part of the URL, if set, will be ignored.
```

Comment:
I think this could be slightly confusing since IAM uses cookies under the hood. Perhaps it is sufficient to say credentials in the URL will be ignored.

Reply:
Done in d4f8c251